tests/unit/Switch-bidi.js

```diff
@@ -0,0 +1,136 @@
+define([
+	"intern!object",
+	"intern/chai!assert"
+], function (registerSuite, assert) {
+	var container, html = "<d-switch id='b1' checkedLabel='\u05d0\u05d1\u05d2' uncheckedLabel='ABC' checked='true' textDir='ltr'></d-switch>" +
+		"<d-switch id='b2' checkedLabel='\u05d0\u05d1\u05d2' uncheckedLabel='ABC' checked='false' textDir='rtl'></d-switch>" +
+		"<d-switch id='b3' checkedLabel='\u05d0\u05d1\u05d2' uncheckedLabel='ABC' checked='true' title='ABC \u05d0\u05d1\u05d2' textDir='auto'></d-switch>";
+
+	registerSuite ({
+		name: "deliteful/Switch (bidi)",
+		setup: function () {
+			moduleRequire = require.config({
```

Comment:
Thinking more about this, I think the only viable solution will be to have bidi tests totally separated with its own intern config (inheriting from the standard one) and another grunt target to launch bidi tests (and maybe yet another one to run both bidi and non bidi).